Lib/test/test_posix.py

```diff
@@ -189,19 +189,29 @@ def test_waitid(self):
             self.assertEqual(pid, res.si_pid)
 
     @unittest.skipUnless(hasattr(os, 'fork'), "test needs os.fork()")
-    def test_register_after_fork(self):
+    def test_register_at_fork(self):
+        with self.assertRaises(TypeError, msg="Positional args not allowed"):
+            os.register_at_fork(lambda: None)
+        with self.assertRaises(TypeError, msg="Args must be callable"):
+            os.register_at_fork(before=2, after_in_child="three",
```

Comment:
It is not clear what argument triggered an error. Use three different invocations with three `assertRaises`.